x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -237,6 +237,8 @@ export const DETECTION_ENGINE_PREPACKAGED_URL =
   `${DETECTION_ENGINE_RULES_URL}/prepackaged` as const;
 export const DETECTION_ENGINE_PRIVILEGES_URL = `${DETECTION_ENGINE_URL}/privileges` as const;
 export const DETECTION_ENGINE_INDEX_URL = `${DETECTION_ENGINE_URL}/index` as const;
+export const DETECTION_ENGINE_INSTALLED_INTEGRATIONS_URL =
```

Comment:
@banderror -- this is as far as I went with plumbing for the `installed_integrations` API. Created this route, and also wired it up to to the `useInstalledIntegrations` hook, so should just start working if you return a `RelatedIntegrationArray` from this route 🙂